Mathlib/AlgebraicGeometry/Scheme.lean

```diff
@@ -761,6 +761,14 @@ lemma stalkMap_germ_apply (U : Y.Opens) (x : X) (hx : f.base x ∈ U) (y) :
       X.presheaf.germ (f ⁻¹ᵁ U) x hx (f.app U y) :=
   PresheafedSpace.stalkMap_germ_apply f.toPshHom U x hx y
 
+/-- If `x` and `y` are inseparable, the stalk maps are isomorphic. -/
+noncomputable def arrowStalkMapIsoOfInseparable {x y : X}
+    (h : Inseparable x y) : Arrow.mk (f.stalkMap x) ≅ Arrow.mk (f.stalkMap y) :=
```

Comment:
I think using `x = y` here makes it easier to use. Since the underlying space of a scheme is sober, no generality is lost.